services/idp/src/App.jsx

```diff
@@ -13,14 +14,23 @@ const LazyMain = lazy(() => import(/* webpackChunkName: "identifier-main" */ './
 
 console.info(`Kopano Identifier build version: ${version.build}`); // eslint-disable-line no-console
 
-const App = (): ReactElement => {
+const App = ({ bgImg }) => {
```

Comment:
Could you check the type again? Might be ReactNode ?

Reply:
Definitely :-)